cmd/signature-v2.go

```diff
@@ -85,8 +85,12 @@ func doesPresignV2SignatureMatch(r *http.Request) APIErrorCode {
 	cred := serverConfig.GetCredential()
 
 	// r.RequestURI will have raw encoded URI as sent by the client.
-	splits := splitStr(r.RequestURI, "?", 2)
-	encodedResource, encodedQuery := splits[0], splits[1]
+	tokens := strings.SplitN(r.RequestURI, "?", 2)
```

Comment:
I also read what `splitStr()` does  irrespective of its name and where it was used.  Of course the code is more readable without it.